tests/conftest.py

```diff
@@ -100,11 +100,11 @@ def network():
     return _get_network()
 
 
-@pytest.fixture(scope='session')
-def global_execution_env():
-    """Network fixture with pre-existing assets in all nodes.
+@pytest.fixture(scope="session")
+def data_envs():
```

Comment:
@jmorel what do you think of this name? Any idea for a better name?

Reply:
I'd rather have the same name for the class returned by the fixture (currently _TestAssets) and the fixture itself (currently data_envs).

`data_env` is pretty nice I think, it the class was renamed in `_DataEnv` and the fixture in `default_data_env` I'd be more than satisfied !

Reply:
Yes, I fully agree.